nixos/modules/system/boot/loader/systemd-boot/systemd-boot-builder.py

```diff
@@ -328,16 +348,23 @@ def main() -> None:
     parser.add_argument('default_config', metavar='DEFAULT-CONFIG', help='The default @distroName@ config to boot')
     args = parser.parse_args()
 
+    subprocess.check_call("@checkMountpoints@")
```

Comment:
What is the effect of this if placed here?

Reply:
Sorry my eyes got tricked by the way github was showing the diff, I thought it was at the end of the `install_bootloader()` function. This is perfectly fine. I'll finish the review tomorrow when I am less tired.